src/WinUIEx/MonitorInfo.cs

```diff
@@ -69,8 +71,13 @@ internal unsafe MonitorInfo(HMONITOR monitor, RECT* rect)
         /// </remarks>
         public Rect RectWork { get; }
 
+        /// <summary>
+        /// Gets if the monitor is the the primary display monitor.
+        /// </summary>
+        public bool IsPrimary { get; }
+
         /// <inheritdoc />
-        public override string ToString() => $"{Name} {RectMonitor.Width}x{RectMonitor.Height}";
+        public override string ToString() => $"{Name} {(IsPrimary ? " Primary" : "")} {RectMonitor.Width}x{RectMonitor.Height}";
```

Comment:
I don't think we should be changing ToString here, What was the reasoning for this?

Reply:
Ok, we can remove this. I really don't know what should be contained in ToString function.